src/aosm/HISTORY.rst

```diff
@@ -5,6 +5,9 @@ Release History
 
 2.0.0b1
 ++++++++
+* Renamed nfdvName to nfdv in CGVs
```

Comment:
If we're updating the changelog, can you also delete line 29, "No changes, building wheel from correct branch"

(I can't comment directly on that line because GitHub is **annoying**.)

Reply:
done